portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,184 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#box {
+    max-width: 2000px;
+    margin:auto;
```

Comment:
I think it's nicer style to have a ' ' after the `:`. Here and elsewhere.

Reply:
' ' added everywhere.